pandas/core/frame.py

```diff
@@ -576,6 +576,15 @@ def _repr_html_(self):
         else:
             return None
 
+    def _repr_latex_(self):
+        """
+        Returns a LaTeX representation for a particular Dataframe.
+        Mainly for use with nbconvert (jupyter notebook conversion to pdf).
+        """
+        is_escaped = get_option("display.latex.escape")
```

Comment:
these options should instead be in ``.to_latex`` (replace the current defaults with ``None``, then use these)
The user can override or use the default options as the defaults.

update the .to_latex doc-string as well